install.sh

```diff
@@ -200,6 +204,38 @@ file_not_grpowned() {
   [[ " $(id -G "$USER") " != *" $(get_group "$1") "*  ]]
 }
 
+# Please sync with 'test-ruby()' in 'Library/Homebrew/utils/ruby.sh' from Homebrew/brew repository.
+test-ruby () {
+  [[ ! -x $1 ]] && { echo "false"; return 1; }
```

Comment:
```suggestion
  if [[ ! -x "$1" ]]; then 
    return 1
  fi
```

Reply:
I'm 👍 for this change (except that there is no need for `"` around `$1`) but I'd like to keep this function identical to its sibling in `utils/ruby.sh` in Homebrew/brew.

Reply:
> except that there is no need for " around $1

```console
$ function cat_file() { cat $1; }
$ cat_file "foo bar"
cat: foo: No such file or directory
cat: bar: No such file or directory
$ function cat_file() { cat "$1"; }
$ cat_file "foo bar"
Hello, world!
```

I believe the quotes are necessary.

Reply:
> I'd like to keep this function identical to its sibling in utils/ruby.sh in Homebrew/brew.

Want to change them both then?

Reply:
> Want to change them both then?

Sure!



> > except that there is no need for " around $1
> 
> ```
> $ function cat_file() { cat $1; }
> $ cat_file "foo bar"
> cat: foo: No such file or directory
> cat: bar: No such file or directory
> $ function cat_file() { cat "$1"; }
> $ cat_file "foo bar"
> Hello, world!
> ```
> 
> I believe the quotes are necessary.

Not inside `[[ ]]`:
```sh
cd $(mktemp -d)
mkdir "dir with spaces"
FILE="dir with spaces/test.sh"
echo "echo Hi" > "$FILE"
chmod u+x "$FILE"
if [[ -x $FILE ]]; then
    echo success
fi
```

The reason is that (from `man bash`):

> Word  splitting  and  pathname  expansion  are  not performed on the words between the [[ and ]];

Reply:
All the exceptions in bash are crazy making. 🤪 😠